ui/admin/app/templates/scopes/scope/targets/target/manage-alias.hbs

```diff
@@ -48,6 +48,7 @@
   <page.body>
     <Form::Alias
       @model={{@model}}
+      @hideTargetId='true'
```

Comment:
waiting on feedback for this, not sure if we should still hide the target id in edit state

Reply:
this has been changed to readonly in new and editable in edit view, see [thread](https://hashicorp.slack.com/archives/C012GTH1C4E/p1713474813278209) here